src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -15595,9 +15595,12 @@ void ComputeTableBodyUsingMovingAvg(EnergyPlusData &state,
         resCellsUsd(LoadCompCol::SensDelay, LoadCompRow::PowerGen) = true;
 
         // DOAS
-        resultCells(LoadCompCol::SensInst, LoadCompRow::DOAS) = state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASHeatAddSeq(timeOfMax);
+        Real64 const mult = state.dataHeatBal->Zone(zoneIndex).Multiplier * state.dataHeatBal->Zone(zoneIndex).ListMultiplier;
+        resultCells(LoadCompCol::SensInst, LoadCompRow::DOAS) =
+            state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASHeatAddSeq(timeOfMax) / mult;
         resCellsUsd(LoadCompCol::SensInst, LoadCompRow::DOAS) = true;
-        resultCells(LoadCompCol::Latent, LoadCompRow::DOAS) = state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASLatAddSeq(timeOfMax);
+        resultCells(LoadCompCol::Latent, LoadCompRow::DOAS) =
+            state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASLatAddSeq(timeOfMax) / mult;
```

Comment:
In the `Estimated Cooling Peak Load Components` table of `Zone Component Load Summary` report the `DOAS Direct to Zone` entry raw is the only one scaled by zone multiplier. Other entries such as `People`, `Lights`, and `Equipment` peak load components are not multiplied by zone multiplier. This fix is intended to fix the issue.  @mjwitte do you are with this proposed change before I proceed further?

Reply:
Yes, `state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASLatAddSeq(timeOfMax)` needs to be divided by `mult`.

Reply:
@mjwitte Thanks.